swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from unicodedata import category
```

Comment:
It looks like this was accidentally imported! It's a good idea to check when you're done coding to make sure VS Code didn't automatically add imports you don't need.